client/src/PlayerLanding/AddPlayer.tsx

```diff
@@ -75,7 +78,7 @@ function AddPlayer() {
               </Dialog.Close>
               <Button
                 onClick={() => {
-                  setPlayer(parsedName);
+                  setMe(parsedName);
```

Comment:
Can't we get rid of all this shit and just setMe after we are done joining the game?

Reply:
oh, not here though, it's rejoin